app/controllers/application_controller.rb

```diff
@@ -1,10 +1,28 @@
 require './config/environment'
 
 class ApplicationController < Sinatra::Base
-
+ register Sinatra::ActiveRecordExtension
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions  
+    set :session_secret, "fwitter secret"
   end
 
+  get '/' do
+    erb :index
+  end  
+  
+
+  helpers do
+
+    def logged_in?
+      !!session[:user_id]
+    end
+
+    def current_user
+      User.find(session[:user_id])
```

Comment:
we could set this to be
```ruby 
@current_user ||= User.find(session[:user_id])
```
so that we aren't querying the database each time if we still have an instance variable set.